source/_components/speedtestdotnet.markdown

```diff
@@ -91,39 +103,17 @@ test. You can turn on manual mode to disable the scheduled speed tests.
 
 ## {% linkable_title Examples %}
 
-In this section, you find some real-life examples of how to use this sensor.
+In this section, you find some real-life examples of how to use this component.
 
 ### {% linkable_title Run periodically %}
 
 Every half hour of every day:
 
 ```yaml
 # Example configuration.yaml entry
-sensor:
-  - platform: speedtest
-    minute:
-      - 0
-      - 30
-    monitored_conditions:
-      - ping
-      - download
-      - upload
-```
-
-### {% linkable_title Run at a specific time %}
-
-Everyday at 12:30AM, 6:30AM, 12:30PM, 6:30PM:
-
-```yaml
-# Example configuration.yaml entry
-sensor:
-  - platform: speedtest
-    minute: 30
-    hour:
-      - 0
-      - 6
-      - 12
-      - 18
+speedtest:
+  update_interval:
+    minutes: 30
     monitored_conditions:
```

Comment:
This indenting is also wrong. monitored_conditions is a peer to update_interval, not a child.